docs/build/guides/conventions/upgrading-contracts.mdx

```diff
@@ -191,60 +213,83 @@ target/wasm32-unknown-unknown/release/soroban_upgradeable_contract_old_contract.
 target/wasm32-unknown-unknown/release/soroban_upgradeable_contract_new_contract.wasm
 ```
 
-## Run the Contract
+## Run the contract
+
+If you have [`stellar-cli`] installed, you can invoke contract functions. Deploy the old contract and install the Wasm for the new contract.
 
-If you have [`stellar-cli`] installed, you can invoke contract functions. Deploy the old contract and install the wasm for the new contract.
+Navigate to to `upgradeable_contract/old_contract`
 
 ```sh
 stellar contract deploy \
-    --wasm target/wasm32-unknown-unknown/release/soroban_upgradeable_contract_old_contract.wasm \
-    --id a
+  --wasm target/wasm32-unknown-unknown/release/soroban_upgradeable_contract_old_contract.wasm  --source 0xkodesage  \
+  --network testnet
 ```
 
-```sh
-stellar contract install \
-    --wasm target/wasm32-unknown-unknown/release/soroban_upgradeable_contract_new_contract.wasm
+- When you deploy a smart contract to a network, you need to specify an identity that will be used to sign the transactions. Change the `0xkodesage` to your own [identity].
+
+[identity]: ../../smart-contracts/getting-started/setup#configure-an-identity
+
+You should see this similar contract ID after you have ran the deploy command:
+
+```
+CAS6FKBXGVXFGU2SPPPJJOIULJNPMPR6NVKWLOQP24SZJPMB76TGH7Y3
+```
+
+Navigate to to `upgradeable_contract/new_contract` and run the following command:
+
+```
+stellar contract install --source-account 0xkodesage  --wasm target/wasm32-unknown-unknown/release/soroban_upgradeable_contract_new_contract.wasm  --rpc-url https://soroban-testnet.stellar.org:443 --network-passphrase "Test SDF Network ; September 2015"   --network testnet
 ```
 
 You should see this Wasm hash from the install command:
 
 ```
-c30c71a382438ed7e56669ba172aa862cc813d093b8d2f45e85b47ba38a89ddc
+aa24c81289997ad815489b29db337b53f284cca5aba86e9a8ae5cef7d31842c2
 ```
 
-You also need to call the `init` method so the `admin` is set. This requires us to setup som identities.
+You also need to call the `init` method so the `admin` is set. This requires us to setup some [identities].
+
+[identities]: ../../smart-contracts/getting-started/setup#configure-an-identity
+
+Given that I have previously setup my own identity, to get the identity address, we run the following command:
 
 ```sh
-stellar keys generate acc1 && \
-stellar keys address acc1
+stellar keys address 0xkodesage
```

Comment:
let's make the name of the identity alice or keep it acc1 here (and wherever else we use it)